databuilder/docs/backends.py

```diff
@@ -9,9 +9,12 @@ def build_backends():
     backends.sort(key=operator.attrgetter("__name__"))
 
     for backend in backends:
+        # TODO: We need to think through the naming here; the things which backends
+        # implement might not be best described as "contracts". At the very least, they
+        # will usually live under `databuilder.tables`.
+        contract_names = [namespace.__name__ for namespace in backend.implements]
+
         yield {
             "name": backend.__name__,
-            # TODO: Re-establish a listing of the tables a given backend supports once
-            # we decide exactly how this is going to work
-            "contracts": [],
+            "contracts": contract_names,
```

Comment:
Does this actually _break_ the docs? Have you made a plan for next steps with Steve?

Reply:
Possibly, and no :) But as above, I didn't want to start conversations with Steve until I had more confidence in these changes.

Reply:
:+1: